app/code/Bulbulatory/Recomendations/Controller/Recommendations/SaveRecommendation.php

```diff
@@ -0,0 +1,88 @@
+<?php
+namespace Bulbulatory\Recomendations\Controller\Recommendations;
+
+use Magento\Framework\Controller\ResultFactory;
+use Bulbulatory\Recomendations\Api\RecommendationRepositoryInterface;
+use Magento\Customer\Model\Session;
+use Magento\Backend\App\Action\Context;
+use Magento\Framework\Math\Random;
+use Bulbulatory\Recomendations\Api\Data\RecommendationInterface;
+use Bulbulatory\Recomendations\Helper\Email;
+use Magento\Framework\UrlInterface;
+use Magento\Customer\Api\CustomerRepositoryInterface;
+
+class SaveRecommendation extends \Magento\Framework\App\Action\Action
+{
+    const CONFIRMATION_URL = 'bulbulatory_recomendations/recommendations/confirmRecommendation';
+
+    protected $recommendationRepository;
+    protected $loggedCustomer;
+    protected $mathRandom;
+    protected $recommendation;
+    protected $email;
+    protected $urlBuilder;
+    protected $customerRepository;
+
+    public function __construct(
+        Context $context,
+        RecommendationRepositoryInterface $recommendationRepository,
+        Session $loggedCustomer,
+        Random $mathRandom,
+        RecommendationInterface $recommendation,
+        Email $email,
+        UrlInterface $urlBuilder,
+        CustomerRepositoryInterface $customerRepository
+    ) {
+        parent::__construct($context);
+        $this->recommendationRepository = $recommendationRepository;
+        $this->loggedCustomer = $loggedCustomer;
+        $this->mathRandom = $mathRandom;
+        $this->recommendation = $recommendation;
+        $this->email = $email;
+        $this->urlBuilder = $urlBuilder;
+        $this->customerRepository = $customerRepository;
+    }
+
+	public function execute()
+	{
+        $post = (array)$this->getRequest()->getPost();
```

Comment:
Jeśli moduł jest wyłączony, to nie powinniśmy dawać możliwości stworzenia rekomendacji. Trzeba by sprawdzić, czy w konfiguracji moduł jest włączony i tylko wtedy utworzyć rekomendację. Podobnie należy sprawdzić, czy użytkownik jest zalogowany.